draco/core/misc.py

```diff
@@ -214,6 +217,58 @@ def finish(self):
         return items
 
 
+class CheckMPIEnvironment(task.MPILoggedTask):
+    """Check that the current MPI environment can communicate
+    across all nodes.
+    """
+
+    timeout = config.Property(proptype=int, default=240)
+
+    def setup(self):
+        import time
+
+        comm = self.comm
+        n = 500000  # Corresponds to a 4 Mb buffer
```

Comment:
```suggestion
        n = 500000  # Corresponds to a 4 MB buffer
```
I think it's 4 megabytes, not megabits.

Reply:
Ah, yes, good catch, I specifically changed that and apparently did it wrong